workflows/iuf/operations/nexus-setup/README.md

```diff
@@ -73,3 +73,29 @@ argo -n argo submit --from workflowtemplate/nexus-docker-upload-template \
   --parameter-file $PRODUCTS_DIR/$PRODUCT/iuf-manifest.yaml \
   --watch
 ```
+
+## `nexus-rpm-upload-template.yaml`
+
+This template will upload content for one or more type "raw" or "yum" repositories into Nexus.
+
+Create the template in Argo by running:
+
+```bash
+argo -n argo template create nexus-rpm-upload-template.yaml
+```
+
+The example below shows how to submit the template using the current required parameters. This example requires the existence
+of the product at `$PRODUCTS_DIR`/`$PRODUCT` and requires the `$NEXUS_SETUP_IMAGE` to be present in Nexus.
+
+```bash
+PRODUCTS_DIR=/admin/rnoska/argo-nexus/nexus-upload/products
+PRODUCT=cos-2.5.38-20221024172946
+NEXUS_SETUP_IMAGE=artifactory.algol60.net/csm-docker/unstable/cray-nexus-setup:0.8.1-20221110002429_3de03c8
+
```

Comment:
Break this up into two blocks, so the command below is easier to copy-paste.

Reply:
I've removed these from the example.